hngi/mobileforce-digital-receipt
# No internet connectivity page

**What does this PR do?**

No Internet Connectivity Page 

**description of Task to be completed?**

This checks if the user is connected to the internet or not and displays a  no internet connection message if the user isn't connected

**How should this be manually tested?**

To test the app, say on main.dart
1- Import, 'digital_receipt/screens/no_internet_connection.dart'
2- call , NoInternetConnectivity()
3-Run the application, navigate through the app
4- Disconnect your internet, the no internet connection page should be displayed

**Any background context you want to provide?**

Any pertinent information that should be considered

**What are the relevant PT Story ID?**

[#0000](https://www.pivotaltracker.com/story/show/0000)

**Screenshots (if appropriate)**
#147
<img width="382" alt="Screen Shot 2020-07-01 at 19 24 38" src="https://user-images.githubusercontent.com/8875216/86297274-c46a4500-bbf2-11ea-9aff-6343324ec62d.png">
